expansion/src/test/java/no/unit/nva/expansion/model/ExpandedDataEntryTest.java

```diff
@@ -196,24 +193,6 @@ public ImportCandidate randomImportCandidate(PublicationContext publicationConte
                    .build();
     }
 
-    @ParameterizedTest(name = "Expanded resource should not lose information for instance type {0}")
-    @MethodSource("publicationInstanceProvider")
-    void shouldReturnExpandedResourceWithoutLossOfInformation(Class<?> instanceType)
-        throws JsonProcessingException, BadRequestException, JSONException {
-        var publication = createPublicationWithoutDoi(instanceType);
-        publication.setFundings(List.of());
-        var expandedResource = fromPublication(uriRetriever, publication);
-
-        var expandedResourceAsJson = expandedResource.toJsonString();
-
-        var regeneratedPublication = objectMapper.readValue(expandedResourceAsJson, Publication.class);
-
-        var jsonOriginal = attempt(() -> objectMapper.writeValueAsString(publication)).orElseThrow();
-        var jsonActual = attempt(() -> objectMapper.writeValueAsString(regeneratedPublication)).orElseThrow();
-
-        JSONAssert.assertEquals(jsonOriginal, jsonActual, false);
-    }
-
```

Comment:
Following test is not needed. This is something we never do in production, and every time working with expansion we have to repair this test.